### PR TITLE
[DOCFIX]Add the bootstrap-conf command snippet in Configuring-Alluxio-with-S3.md of the Chinese version according to the English version for Alluxio 1.1

### PR DESCRIPTION
**Pull request by GrandpaXun**
https://alluxio.atlassian.net/browse/ALLUXIO-1981

### CODE REVIEW

**Comment by ifcharming** [comment]
ok to test

File: docs/cn/Configuring-Alluxio-with-S3.md [hunk 1]
```diff
@@ -12,7 +12,13 @@ priority: 0
 
 首先，本地要有Alluxio二进制包。你可以自己[编译Alluxio](Building-Alluxio-Master-Branch.html)，或者[下载二进制包](Running-Alluxio-Locally.html)
 
-然后，如果你还没有配置文件，由template文件创建配置文件：
+然后，如果你还没有进行这项配置，那么用‘bootstrap-conf’命令创建你的配置文件
```
Comment (ifcharming) [inline docs/cn/Configuring-Alluxio-with-S3.md]: please use "`" instead of "‘" for the Markdown syntax.

**Comment by ifcharming** [comment]
thanks @GrandpaXun for using the right github account! 
I left one minor comment, otherwise LGTM

File: docs/cn/Configuring-Alluxio-with-S3.md [hunk 2]
```diff
@@ -12,7 +12,13 @@ priority: 0
 
 首先，本地要有Alluxio二进制包。你可以自己[编译Alluxio](Building-Alluxio-Master-Branch.html)，或者[下载二进制包](Running-Alluxio-Locally.html)
 
-然后，如果你还没有配置文件，由template文件创建配置文件：
+然后，如果你还没有进行这项配置，那么用`bootstrap-conf`命令创建你的配置文件
+
+例如，如果你在本地机器上运行Alluxio，ALLUXIO_MASTER_HOSTNAME’应该设置为`localhost`。
```
Comment (ifcharming) [inline docs/cn/Configuring-Alluxio-with-S3.md]: can you please also update ALLUXIO_MASTER_HOSTNAME with the right symbol?

**Comment by apc999** [comment]
Thanks for the iterations and LGTM!

**Pull request merged**